### DISCUSSION TITLE
Happy New Year Everyone 🎉

### DISCUSSION THREAD

**Discussion by ChlorinePentoxide** [openAOD/Join#40] [General]
I, as part of the PatternHouse Organization, wish all our members a very happy new year. Hope every one has a colourful, productive, energetic and eventful year ahead of us. 

A whole new feature set has been planned for the upcoming year,

- A change-aware atomic build system to enable unified fast testing of incoming source code
- A blog system for the website

Hope we can receive more contributions from our new members. 

Regards,
ChlorinePentoxide
PatternHouse Maintainer

**Comment by ChlorinePentoxide** [comment]
@aryashah2k Hopefully you can spread the word on the Discord and Gitter channels.

**Comment by aryashah2k** [comment]
I wish a Happy New Year to all our members and a special thanks to you @ChlorinePentoxide , trust me buddy, majority of this project would not be at this stage of development if it was'nt you!
I am passing this screenshot on the Disocrd channel for all our members through SWOC and yeah! The new year has a lot in store for you, me and this project. 
Thanks for sticking through thick and thin with me on this one buddy! @ChlorinePentoxide

**Comment by ChlorinePentoxide** [comment]
Thank you @aryashah2k. I am grateful to be a part of this project 😃
